Dockerfiles/agent/windows/amd64/Dockerfile

```diff
@@ -6,11 +6,17 @@ ARG BASE_IMAGE=mcr.microsoft.com/windows/servercore:1809
 #ARG BASE_IMAGE=mcr.microsoft.com/powershell:nanoserver-1809
 FROM ${BASE_IMAGE} 
 
+ARG WITH_JMX="false"
+
 LABEL maintainer "Datadog <package@datadoghq.com>"
 
+USER ContainerAdministrator

```

Comment:
This is redundant as it's default value no?

Reply:
It's the default on the `servercore` image, but not on other images I've been using (eg: the servernano + powershell image). I think it's better to keep it explicit.

Reply:
Okay, so what's the default user in `nano`, is it a user with limited rights?

Reply:
On the image I used (official powershell image on nano, since nano doesn't have powershell by default) was `ContainerUser`.

Reply:
Okay, got it!